frontend/lib/src/components/widgets/DataFrame/hooks/useDataLoader.ts

```diff
@@ -82,7 +80,6 @@ function useDataLoader(
         const arrowCell = data.getCell(originalRow + 1, originalCol)
         return getCellFromArrow(column, arrowCell, data.cssStyles)
       } catch (error) {
-        logError(error)
```

Comment:
Drive-by cleanup: this logging isn't really necessary and this case is also triggered ins situations where its totally ok that it runs into this issue (the data gets updated and the new data has less rows).